src/ZoneServer/Scripting/AI/AiScript.Routines.cs

```diff
@@ -238,16 +260,18 @@ protected IEnumerable Follow(ICombatEntity followTarget, float minDistance = 50,
 					// recreate it on the other side, from the summoning
 					// script. All this could need a clean up.
 
-					if (!ZoneServer.Instance.Conf.World.BlueOrbFollowWarp)
+					if (!ZoneServer.Instance.Conf.World.BlueOrbFollowWarp && !ZoneServer.Instance.Conf.World.BlueOrbPetSystem)
```

Comment:
Wouldn't it be confusing if one conf option disables another implicitly? I realize you wouldn't typically want your "blue orb pets" to stay behind, but isn't that an choice the user should make? Maybe someone wants to limit BO pets to one map, in which case they could enable the pet option but leave follow warp disable.

Reply:
It actually does still respect the follow warp setting with the pet system turned on, the effect of this is in summoning.cs.  Basically, if the pet system is turned off, when you go through a warp, your monster just stays there until it despawns.  This is not a valid option if the pet system is turned on as the monster has no time limit and will just get stuck there forever, so instead, if follow warp is turned off, it removes your monster (as per what would normally happen with follow warp turned on), but it doesn't respawn it on the other side, forcing you to recall it manually.

Reply:
Right, but if I were a user and I turned "blue orb pets" on and "follow warp" off I might expect them to stay behind. If that's not going to be the case this behavior should be documented in the conf files to avoid confusion.

Reply:
That's a good point, I've added more detail to the description now.